apps/settings/js/panels/root/panel.js

```diff
@@ -158,3 +133,29 @@ define(function(require) {
     });
   };
 });
+/**
+ * Update the sim and NFC related items
+   based on mozMobileConnections and Device.
+ */
+
+var updateSimItemsAndNfc = function rph_refrehsSimItems() {
```

Comment:
Ok, but:
 1. this code still is not required for 'root' panel, since you are moving appropriate UI to separate panels (as I can see from initial commit for 'Settings' they are: ``Privacy_security`` and ``Network``) you shall move this code to .js modules for appropriate panels. Creation and loading of js for panels is pretty well described in README.md file in settings folder.
 2. in the mozilla's version of this functionality 'NFC' part was done independently from sim-related ones.
 3. BTW, I failed to find the html item with ``id == simCardManager-settings`` is this item expected to be created dynamically?
 4. Technically it's possible to define several independent components in single .js file, e.g. already mentioned startup.js, but this is not the case.